.pre-commit-config.yaml

```diff
@@ -49,9 +49,16 @@ repos:
         entry: cargo test --package sargon --test vectors
         pass_filenames: false
 
-      - id: uniffi bindgen tests
-        name: uniffi bindgen tests
+      - id: swift tests
+        name: swift tests
         language: system
-        types_or: [file, rust, swift]
-        entry: cargo test --package sargon --test uniffi
+        types_or: [rust, swift]
+        entry: ./scripts/ios/test.sh
+        pass_filenames: false
+
+      - id: kotlin tests
+        name: kotlin (jvm) tests
+        language: system
+        types_or: [rust, kotlin]
```

Comment:
Types Or is a nice filter, if we make changes in README or other non Rust, non Kotlin files (or just Swift files) the precommit will not run the JVM tests